modules/benchmark/R/metric.scatter.R

```diff
@@ -2,17 +2,15 @@
 ##' @title metric.scatter.plot
 ##' @export
 ##' @param dat
+##' @param var
 ##' 
 ##' @author Betsy Cowdery
-
-metric.scatter.plot <- function(dat, var){
+metric.scatter.plot <- function(dat, var) {
   
   library(ggplot2)
   
-  ggplot(data = dat) + 
-    geom_point(aes(x=model,y=obvs), size=4) + 
-    geom_abline(slope=1,intercept=0, colour = "#666666", size=2, linetype = 2, lineend = "round")  
+  ggplot(data = dat) + geom_point(aes(x = model, y = obvs), size = 4) + geom_abline(slope = 1, intercept = 0, 
+    colour = "#666666", size = 2, linetype = 2, lineend = "round")
```

Comment:
I don't mean be difficult but this seems to be making the ggplot command harder to read. As opposed to easier.